converter/internal/otelcolconvert/testdata/tail_sampling.river

```diff
@@ -0,0 +1,229 @@
+otelcol.receiver.otlp "default" {
```

Comment:
This is taken right from the examples from otel: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md